src/adversarial/neurify.jl

```diff
@@ -26,29 +26,45 @@ Sound but not complete.
 """
 
 @with_kw struct Neurify
-    max_iter::Int64     = 1000
+    max_iter::Int64     = 10
     tree_search::Symbol = :DFS # only :DFS/:BFS allowed? If so, we should assert this.
+    model = Nothing()
+    optimizer = GLPK.Optimizer
+    splitted = Set() # To prevent infinity loop. Not the best solution, need to be modified in the future.
```

Comment:
Lets go with `split` or `splits` or `split_intervals` or something else depending on the spirit of how you meant it, since `splitted` isn't a word

Reply:
`splitted` is a set to prevent splitting a ReLU node more than one time. What do you think is a proper name?

Reply:
I change it to `splits`